wordle-web/pages/game.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="500">
+    <v-card-item>
+      <v-card-title> Would you want to light up your day? </v-card-title>
+      <v-card-subtitle> (y/n) </v-card-subtitle>
+    </v-card-item>
+
+    <v-text-field
+      v-if="!jokeAsked"
+      v-model="input"
+      label="Obviously Yes:)"
+      outlined
+      clearable />
+    <v-text-field
+      v-else
+      v-model="punchline"
+      label="Why don't scientists trust atoms?"
+      outlined
+      clearable />
+
+    <v-card-actions>
+      <v-spacer />
+      <v-btn
+          color="blue"
+          variant="elevated"
+          elevation="8"
+          @click="jokeAsked ? checkPunchline() : askForJoke()">
+          Enter
+      </v-btn>
+
+    </v-card-actions>
+    <v-card-text>
+      {{ output }}
+    </v-card-text>
+    <v-card-text>
+      <NuxtLink to="/" external> Home page </NuxtLink>
```

Comment:
might want to have the nav bars smaller rather than having them stretch across the screen

Reply:
We changed it, thank you for the advice.